lib/core/connector_settings.rb

```diff
@@ -7,11 +7,9 @@
 # frozen_string_literal: true
 
 require 'active_support/core_ext/hash/indifferent_access'
-require 'active_support/core_ext/object/blank'
```

Comment:
I think that's required for `.empty?` that you use on line 106, might be wrong tho

Reply:
I don't see any error when I tested it locally. I think `.empty?` is a method of string in core ruby?

Reply:
You are right, sorry for the noise :)